indico/modules/events/fields.py

```diff
@@ -267,7 +267,8 @@ def __init__(self, *args, **kwargs):
 class ReviewQuestionsField(MultipleItemsField):
     def __init__(self, *args, **kwargs):
         self.extra_fields = kwargs.pop('extra_fields', [])
-        self.fields = [{'id': 'text', 'caption': _("Question"), 'type': 'text', 'required': True}] + self.extra_fields
```

Comment:
No strong opinion but IMO this looks better since it doesn't wrap inside the dict
```
        self.fields = ([{'id': 'text', 'caption': _("Question"), 'type': 'textarea', 'required': True}] + 
                       self.extra_fields)
```